### ISSUE TITLE
🚀 Feature Request: Global Font Preview in Editor Not Helpful

### ISSUE THREAD

**Issue by slideri812** [elementor/elementor#19117]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

Hi,
I am a designer and a programmer. I understand why to some people it is helpful to see a preview of the font style when selecting a Global style to apply to an element.

I personally rely on textual descriptions more than a visual. 

When I have 30, 40 or 50 styles to make it easy to manage changes across large sites with lots of different feature elements my descriptions go like this:

- Header - Page Title Tahoma 50px
- Header - Page Subtitle Tahoma 35px
- Page - Title Large Bold Helvetica 35px
- Page - Subtitle Large Mono 30px
- Page - Subtitle Thin 20px
- Section - Title Large Bold 30px
- Section - Title Small Tahoma 20px

Having them previewed in their actual font size means I can't really see them to read them.
I'm sure I'm not the only developer struggling with this.

Thanks

### Describe the solution you'd like

So my request is that the Global Style names in the preview (see attached) be scaled down and not be a direct reflection of the font size.

### Describe alternatives you've considered

Maybe being able to also manage the Global Fonts from outside the Page Editor would be nice.

### Additional context

<img width="233" alt="global font example" src="https://user-images.githubusercontent.com/4176159/178090770-cb22f0dc-d795-4891-bdb9-0d9048cb38ca.png">

**Comment by 79ho3ein** [comment]
I quite agree. Font preview in elementor editor is not really a cool idea.

**Comment by pauserratgutierrez** [comment]
I agree with everyone. The ones who need preview and the ones who don't need it. So my solution is to make the preview an option so the user can activate it or not

**Comment by danijelBalaban** [comment]
@rami-elementor is there a quick workaround for this before it get's adopted in an official release? I'm working on a project which has a lot of custom font sizes and the font preview really makes it difficult to use for sizes larger than 3 REM. Any help is much appreciated.

**Comment by SolunaCG** [comment]
I´d love to have this changed, too! Especially with very big fonts it´s impossible to know which one is which. I´m working on a more complex website which has many huge headings on it but because of how big they are it´s impossible to read them... 

As an addition to what´s already been said I´d love to have the "Global Fonts" window be wider in general so that even longer font names are still completely visible. Or maybe there´s even a way to make it a resizable window (just like the "Navigator"-window which you can already resize). 

But a wider window in general in combination with a description of the fonts just like @slideri812 already wrote would probably be enough.

**Comment by slideri812** [comment]
Hi, no, no resolve to this issue. I had a site with about 50 fonts
(everything centralized to make site wide changes easy). Still no
addressing this is the latest update. Sorry I have no work around. That
would take someone smarter than me code wise to do.

**Comment by SolunaCG** [comment]
Well, what works for me (at least temporarily) is to edit the "editor.min.css" file with the Wordpress Theme File Editor. If you just add `.e-global__preview-item{font-size:16px !important;}` (or whatever font-size you´d like to have) at the end of the file it seems to work.
I´m not a programmer at all and this solution is probably just temporary until the next Elementor update. For me at least a better solution from Elementor would still be really appreciated! 
Not sure if this helps someone... :-)
